### ISSUE TITLE
Building image fails with selected SDK being Java 16

### ISSUE THREAD

**Issue by petertrr** [saveourtool/save-cloud#278]
The `opnjdk:16` probably uses a different base image than 11. Other images should be checked too:
```
com.github.dockerjava.api.exception.DockerClientException: Could not build image: The command '/bin/sh -c apt-get update && env DEBIAN_FRONTEND="noninteractive" apt-get install -y libcurl4-openssl-dev tzdata && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 127
```
This is because `openjdk:16` is an image based on Oracle Linux Server, which uses `rpm` as a package manager; while `openjdk:11` is based on Debian. I believe, there are Debian-based images for newer versions too, though they require a special tag format.

**Comment by akuleshov7** [comment]
Do we really need this?

**Comment by petertrr** [comment]
> Do we really need this?

Yes, or do you suggest to allow only Java 8 and 11?

**Comment by akuleshov7** [comment]
At least by for now let's not touch it :)

**Comment by petertrr** [comment]
> At least by for now let's not touch it :)

We should also disable these versions in our dropdown selection in order not to confuse the users

**Comment by petertrr** [comment]
Under #1030 we should ensure it at least builds for Java 8, 11 and 17

**Comment by petertrr** [comment]
The reason is that `openjdk` images starting from version 12 are using Oracle Linux as a base, which is not Debian-based. This issue will be closed by #1030 by swithing to Eclipse Temurin and keeping only LTS versions
